### PR TITLE
bzt 1.13.5

### PR DESCRIPTION
**Pull request by greyfenrir**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

**Comment by SMillerDev** [comment]
Thanks @greyfenrir! Without contributions like yours it'd be impossible to keep homebrew going! 👍 🎉

File: Formula/bzt.rb [hunk 1]
```diff
@@ -3,9 +3,9 @@ class Bzt < Formula
 
   desc "BlazeMeter Taurus"
   homepage "https://gettaurus.org/"
-  url "https://files.pythonhosted.org/packages/source/b/bzt/bzt-1.13.4.tar.gz"
-  sha256 "b3d4a81f757f200d76c0dfc18fe1731a7e29a9969a3691355bb3dfb1a7af3e75"
-  head "https://github.com/Blazemeter/taurus.git"
+  url "https://files.pythonhosted.org/packages/source/b/bzt/bzt-1.13.5.tar.gz"
+  sha256 "3b19f007e40961241f6bc084cd44f45a3875cdf51a8e967c9cf4a7c7ed62185e"
+  head "https://github.com/greyfenrir/taurus.git"
```
Comment (lembacon) [inline Formula/bzt.rb]: I assume that the change of `HEAD` is not intentional, right?

Comment (greyfenrir) [reply Formula/bzt.rb]: my bad. thanks for note

**Comment by lembacon** [comment]
Thanks @greyfenrir!

**Pull request closed without merging**